lib/cc/engine/duplication.rb

```diff
@@ -43,14 +44,12 @@ def languages_to_analyze
       end
 
       def languages
-        config_languages = engine_config.
-          fetch("config", {}).
-          fetch("languages", DEFAULT_LANGUAGE)
+        languages = engine_config.languages
 
-        if config_languages.is_a?(Hash)
-          config_languages.keys
+        if languages.empty?
+          DEFAULT_LANGUAGE
```

Comment:
Or, what about having `DEFAULT_LANGUAGE` just return `"ruby"` instead of an array?